packages/react/src/components/PageHeader/PageHeader-test.js

```diff
@@ -226,6 +226,29 @@ describe('PageHeader', () => {
   });
 
   describe('PageHeader.TabBar component api', () => {
+    beforeEach(() => {
+      jest.resetModules();
+      // mock window.matchMedia directly
+      Object.defineProperty(window, 'matchMedia', {
+        writable: true,
+        value: jest.fn().mockImplementation((query) => ({
+          matches: false,
+          media: query,
+          onchange: null,
+          addListener: jest.fn(),
+          removeListener: jest.fn(),
+          addEventListener: jest.fn(),
+          removeEventListener: jest.fn(),
+          dispatchEvent: jest.fn(),
+        })),
+      });
```

Comment:
Does this work with the more condensed example used elsewhere with matchMedia?
```suggestion
      jest.spyOn(hooks, 'useMatchMedia').mockImplementation(() => true);
```